### ISSUE TITLE
cannot use stat.Fstypename[:] (type []byte) as type []int8 in argument to common.IntToString

### ISSUE THREAD

**Issue by dpogorzelski** [hashicorp/nomad-skeleton-driver-plugin#10]
Hello all, during dependency upgrade process I came across this issue, (Darwin only it seems):

```bash
../../../go/pkg/mod/github.com/hashicorp/gopsutil@v2.18.12+incompatible/disk/disk_darwin.go:117:43: cannot use stat.Fstypename[:] (type []byte) as type []int8 in argument to common.IntToString
```

**Comment by dpogorzelski** [comment]
I cannot open an issue on the actual gopsutil repo as that tab is disabled it seems

**Comment by lgfa29** [comment]
Hi @dpogorzelski, sorry for missing this issue! 

Is this still a problem? Which dependency were you trying to update?

**Comment by angrycub** [comment]
Fixed by #16 - Switching away from the fork to shirou/gopsutil/v3@3.22.6 includes the fix from https://github.com/shirou/gopsutil/pull/970.
